main/com/collections/Methods.java

```diff
@@ -137,4 +137,91 @@ public boolean canBePalindrome(String s) {
 
         return arraysSum;
     }
+
+    private void reverseArray(int[] array) {
+        for (int i = 0; i < array.length / 2; i++) {
+            int temp = array[i];
+            array[i] = array[array.length - i - 1];
+            array[array.length - i - 1] = temp;
+        }
+    }
+
+    public int[] sumTwoIntArrays1(int[] array1, int[] array2) {
+
+        // reverse Arrays with private method
```

Comment:
You can avoid such comment, name of the method - **reverseArray** - clearly describe what's happening here.